ui/pages/setup_secure.tsx

```diff
@@ -44,29 +40,30 @@ const SetupSecure: NextPage = () => {
     // Otherwise we initialize it
     kratos
       .createBrowserSettingsFlow({
-        returnTo: returnTo ? String(returnTo) : undefined,
+        returnTo: return_to
+          ? return_to.toString()
+          : window.location.pathname.replace(`setup_secure`, `setup_complete`),
```

Comment:
```suggestion
          : window.location.pathname.replace('setup_secure', 'setup_complete'),
```
These are static strings, no need for a template